### ISSUE TITLE
Update documentation of pending beatmap limits

### ISSUE THREAD

**Issue by Walavouchey** [ppy/osu-wiki#7456]
Currently we have this on the wiki page:

![image](https://user-images.githubusercontent.com/36758269/172058574-7a3958b6-1344-41db-b708-d7cb75d61131.png)

Based on the osu!supporter promo page, the supporter limit for pending beatmaps is wrong:

<img src="https://user-images.githubusercontent.com/36758269/172058591-a5285c34-9fd4-4c42-aeaf-a05d72ef4402.png" height="300" />

<img src="https://user-images.githubusercontent.com/36758269/172058612-4d1bfca1-fdde-4298-bb52-f0468a718c98.png" height="200" />

According to <https://osu.ppy.sh/community/forums/topics/1388182?n=34>, the non-supporter one has since been increased:

![image](https://user-images.githubusercontent.com/36758269/172058852-df11c84b-2cff-4cbd-a538-59df6948fb92.png)

In short, it should be `2 + min(ranked beatmaps, 6)` (max 8) for non-supporter and `8 + min(ranked beatmaps, 12)` (max 20) for supporters, and the non-supporter limit should be updated on the promo page as well

It is also ambiguous whether the "(up to N)" on the promo page refers to the maximum total or the maximum bonus, but I'm assuming the latter

cc @peppy

**Comment by peppy** [comment]
Yeah, this will need to be updated in `osu-web` strings.

> It is also ambiguous whether the "(up to N)" on the promo page refers to the maximum total or the maximum bonus, but I'm assuming the latter

Correct, but I think it should be reworded to remove any ambiguity.

**Comment by TicClick** [comment]
please move it to the osu-web repo when #7484 is merged

**Comment by peppy** [comment]
I've updated these to match expectations. Will be visible in the next osu-web deploy.
